drivers/hmis/spec/requests/hmis/lookup_client_enrollment_spec.rb

```diff
@@ -0,0 +1,141 @@
+require 'rails_helper'
+require_relative 'login_and_permissions'
+require_relative 'hmis_base_setup'
+
+RSpec.describe Hmis::GraphqlController, type: :request do
+  before(:all) do
+    cleanup_test_environment
+  end
+  after(:all) do
+    cleanup_test_environment
+  end
+
+  include_context 'hmis base setup'
+
+  let!(:c1) { create :hmis_hud_client_complete, data_source: ds1, user: u1 }
+  let!(:e1) { create :hmis_hud_enrollment, data_source: ds1, project: p1, client: c1, user: u1 }
+  let!(:income_benefit) { create :hmis_income_benefit, data_source: ds1, client: c1, user: u1, enrollment: e1 }
+  let!(:health) { create :hmis_health_and_dv, data_source: ds1, client: c1, user: u1, enrollment: e1 }
+  let!(:disability) { create :hmis_disability, data_source: ds1, client: c1, user: u1, enrollment: e1 }
+  let!(:s1) { create :hmis_hud_service, data_source: ds1, client: c1, enrollment: e1, user: u1 }
+  let!(:a1) { create :hmis_hud_assessment, data_source: ds1, client: c1, enrollment: e1, user: u1 }
+  let!(:ev1) { create :hmis_hud_event, data_source: ds1, client: c1, enrollment: e1, user: u1 }
+
+  before(:each) do
+    hmis_login(user)
+  end
+
+  let(:client_query) do
+    <<~GRAPHQL
+      query Client($id: ID!) {
+        client(id: $id) {
+          #{scalar_fields(Types::HmisSchema::Client)}
+          enrollments {
+            nodesCount
+            nodes {
+              #{scalar_fields(Types::HmisSchema::Enrollment)}
```

Comment:
Util for including all scalar fields in the response. This is a simple thing to help surface errors if any field fails to resolve, without needing to list them all out